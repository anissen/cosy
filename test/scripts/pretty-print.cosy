let b=3{
let a=b+4
fn g(){return fn(x){return(a+x)*2}}print g()(2)}
let f = fn()Fn(Num)Num{return fn(x Num)Num{return x+2}}
f()
