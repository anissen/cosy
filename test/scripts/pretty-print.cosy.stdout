var b = 3
{
  var a = b + 4
  fn g() {
    return fn(x) {
      return (a + x) * 2
    }
  }
  print g()(2)
}
var f = fn() Fn(Num) Num {
  return fn(x Num) Num {
    return x + 2
  }
}
f()
